### PR TITLE
zkSend SDK Changes

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

Describe the changes or additions included in this PR.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

**Review by hayes-mysten — APPROVED** [review]

File: sdk/docs/pages/dapp-kit/stashed.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Stashed Integration
+
+dApp Kit provides out-of-the-box opt-in support for the [Stashed Wallet](../zksend/dapp.mdx).
```
Comment (ronny-mysten) [inline sdk/docs/pages/dapp-kit/stashed.mdx:3]: nit: Wallet is lower case everywhere? else.

File: sdk/docs/pages/zksend/dapp.mdx [hunk 2]
```diff
@@ -4,43 +4,43 @@ import { Callout } from 'nextra/components';
 
 <Callout type="info">
 	If you are using `dapp-kit`, you do not need to install any additional packages to integrate with
-	the zkSend wallet. [Read how to integrate with dapp-kit.](../dapp-kit/zksend.mdx)
+	the Stashed wallet. [Read how to integrate with dapp-kit.](../dapp-kit/stashed.mdx)
```
Comment (ronny-mysten) [inline sdk/docs/pages/zksend/dapp.mdx:7]: I'm sure one exists, but asking about redirect anyway.

File: sdk/docs/pages/zksend/dapp.mdx [hunk 3]
```diff
@@ -4,43 +4,43 @@ import { Callout } from 'nextra/components';
...
+Using the zkSend SDK, you can allow users to connect to the Stashed wallet from your dApp. The
+wallet is provided through the [Wallet Standard](https://docs.sui.io/standards/wallet-standard), and
+should appear automatically in your existing wallet connection UI.
 
 ## Considerations
 
-- zkSend only supports mainnet at this time.
-- Users will only be able to connect their zkLogin wallet from zkSend.
-- The zkLogin account will be managed by zkSend, you do not need to set up any OAuth providers.
+- Stashed only supports mainnet at this time.
```
Comment (ronny-mysten) [inline sdk/docs/pages/zksend/dapp.mdx:16]:
```suggestion
- Stashed only supports Mainnet at this time.
```
Style guide states this form, but better to be consistent with rest of this site if this is the way it's done.

File: sdk/docs/pages/zksend/dapp.mdx (hunk 2 above)
Comment (Jordan-Mysten) [reply sdk/docs/pages/zksend/dapp.mdx:7]: Oh good call let me configure one

**Pull request merged**